src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -2886,4 +2886,22 @@ public override void Accept(OperationVisitor visitor)
 
         protected override OperationKind ExpressionKind => OperationKind.None;
     }
+
+    /// <summary>
+    /// Providing implementation as OperationKind.None. This implementation is sufficient because the node doesn't survive initial binding.
+    /// </summary>
+    internal partial class DeconstructionLocalPendingInference
+    {
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitNoneOperation(this);
```

Comment:
Done.
`OutVarLocalPendingInference` does implement OperatiionKind.None however, even though it also does not survive initial binding. Should it be changed too?